### ISSUE TITLE
setFFmpegPath on Windows requires running twice.

### ISSUE THREAD

**Issue by tmatthias29** [scikit-video/scikit-video#38]
This is almost certainly related to [Issue 21](https://github.com/scikit-video/scikit-video/issues/21), but on Windows, I have to run skvideo.setFFmpegPath(r'Path\to\binary') twice for it to work.  Running it once, I'll receive a warning, and further skvideo calls will fail.  However, running that same command a second time seems to solve the issue.

```python
(toolchain) D:\bppci\ectests\bpp\main>python
Python 2.7.13 (v2.7.13:a06454b1afa1, Dec 17 2016, 20:42:59) [MSC v.1500 32 bit (Intel)] on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> import skvideo
>>> skvideo.setFFmpegPath(r'D:\bbpci\ectests\bpp\main\bin\ffmpeg\windows\stable')
D:\bppci\ectests\bpp\main\toolchain\lib\site-packages\skvideo\__init__.py:310: UserWarning: ffmpeg/ffprobe not found in path: D:\bbpci\ectests\bpp\main\bin\ffmpeg\windows\stable
  warnings.warn("ffmpeg/ffprobe not found in path: " + str(path), UserWarning)
>>> skvideo.setFFmpegPath(r'D:\bbpci\ectests\bpp\main\bin\ffmpeg\windows\stable')
>>> skvideo.getFFmpegPath()
'D:\\bbpci\\ectests\\bpp\\main\\bin\\ffmpeg\\windows\\stable'
>>>
```

**Comment by tmatthias29** [comment]
A note on this, I've tried combinations of `os.path.normpath` and the rawstring character, but that doesn't seem to make a difference.   Also, if you call `skvideo.getFFmpegPath()` after the first attempt, it does return the path, but it seems like skvideo isn't working correctly after.

**Comment by beyondmetis** [comment]
I pushed a fix where I use os.path.join to form the full execution path. You were getting a '/' appended to the end of your path string, which likely made windows upset. I'm surprised that this worked for other people windows users so far.

Let me know if it works or doesn't work, and we can keep digging.

**Comment by tmatthias29** [comment]
Looks like this is fixed in 1.1.8.  Thanks!
